roles/zabbix_proxy/defaults/main.yml

```diff
@@ -112,7 +112,7 @@ zabbix_proxy_vmwarefrequency: 60
 # Zabbix API stuff
 zabbix_api_server_host: localhost
 zabbix_api_use_ssl: false
-zabbix_api_server_port: 80
+# zabbix_api_server_port: 80
```

Comment:
I like having all the vars in defaults/main.yml, is there a reason you want to remove this line?

Reply:
See comment above